client/src/components/PostedOn.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Typography } from "@mui/material";
+import { format } from "date-fns";
+import { useTheme } from "@emotion/react";
+
+//assuming the format of the props.date is YYYY-MM-DD
+const PostedOn = (props) => {
```

Comment:
It will be best to have props.date be a date object and for the conversion to happen in the calling/parent component.